src/SuperstateToken.sol

```diff
@@ -452,6 +450,31 @@ abstract contract SuperstateToken is
         _setMaximumOracleDelay(_newMaxOracleDelay);
     }
 
+    /**
+     * @notice The ```updateStablecoinConfig``` function sets the configuration fields for accepted stablecoins for onchain subscriptions
+     * @dev Requires msg.sender to be the owner address
+     * @param stablecoin The address of the stablecoin
+     * @param newSweepDestination The new address to sweep stablecoin subscriptions to
+     * @param newFee The new fee in basis points to charge for subscriptions in ```stablecoin```
+     */
+    function setStablecoinConfig(address stablecoin, address newSweepDestination, uint96 newFee) external {
+        if (newFee > 10) revert FeeTooHigh(); // Max 0.1% fee
```

Comment:
Is the max fee something that we want to bake into the contract?

Reply:
rob told me it will never ever be higher than this